scripts/package-build/xen-guest-agent/package.toml

```diff
@@ -0,0 +1,22 @@
+[[packages]]
+name = "xen-guest-agent"
+commit_id = "0.4.0"
+scm_url = "https://gitlab.com/xen-project/xen-guest-agent"
+
+
+build_cmd = """
+sudo apt-get -y install --no-install-recommends libclang-dev libxen-dev
+# Install rust
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | bash -s -- -y --default-toolchain stable --profile default --no-modify-path
+
+# Set PATH for Cargo
+export PATH="$HOME/.cargo/bin:$PATH"
+rustup update
+cd xen-guest-agent
+cargo update
+cargo install cargo-deb
+
+# Build deb
+cargo deb
```

Comment:
There is a peculiarity there. By default, the `xenstore-rs` crates links against the system `libxenstore.so`, but the Debian packaging doesn't have that dependency so building with default flags will result in a broken installation, unless libxenstore is installed by hand.

I suppose `cargo deb -F static` may help.

See my MR in the upstream for details: https://gitlab.com/xen-project/xen-guest-agent/-/merge_requests/86